pkg/kubelet/gpu/nvidia/nvidia_gpu_manager.go

```diff
@@ -101,8 +102,7 @@ func (ngm *nvidiaGPUManager) Start() error {
 	if err := ngm.discoverGPUs(); err != nil {
 		return err
 	}
-	// It's possible that the runtime isn't available now.
-	ngm.allocated = ngm.gpusInUse()
+
```

Comment:
but as the PR described, the number of activePods always remains 0 in the Start() function. And as the ngm.allocated has been initialized, it takes no chance to get the right data anymore.
If we don't erase this line from code, we have to call the Start() function late after the pods have been recovered so the activePods will not be 0 in the gpusInUse() function.